libs/SalesforceReact/AndroidManifest.xml

```diff
@@ -3,6 +3,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     android:versionCode="82"
     android:versionName="11.1.0.dev">
-
-    <application />
+    
+    <application>
+        <activity android:name=".ui.SalesforceReactActivity"
+            android:enableOnBackInvokedCallback="false" />
```

Comment:
[RN will eventually support this but it isn't in yet](https://github.com/facebook/react-native/issues/34529).  We can remove this when we update to whatever version that may be.